src/signals/incident/definitions/wizard-step-2-vulaan/openbaarGroenEnWater.ts

```diff
@@ -36,27 +40,27 @@ export const controls = {
       icons: [
         {
           id: 'oak',
-          icon: caterpillarIcons.oak,
+          icon: oakUrl,
```

Comment:
Shouldn't the prop name be `iconUrl`? For consistency sake.

Reply:
sure